From e7e702f88f289093719fb9ef913217bbb91788a8 Mon Sep 17 00:00:00 2001
From: Johan Pascal <johan.pascal@belledonne-communications.com>
Date: Mon, 2 Mar 2015 21:15:57 +0100
Subject: [PATCH 3/4] Enable AES256 for SRTP after ZRTP negociation but keep
 AES128 as default - setting in sip section srtp_crypto_suite in the
 configuration file + update ms

---
 coreapi/misc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/coreapi/misc.c b/coreapi/misc.c
index 8dc6cf8..6071e6b 100644
--- a/coreapi/misc.c
+++ b/coreapi/misc.c
@@ -1533,7 +1533,7 @@ void linphone_core_set_tone(LinphoneCore *lc, LinphoneToneID id, const char *aud
 }
 
 const MSCryptoSuite * linphone_core_get_srtp_crypto_suites(LinphoneCore *lc){
-	const char *config=lp_config_get_string(lc->config,"sip","srtp_crypto_suites","AES_CM_128_HMAC_SHA1_80, AES_CM_128_HMAC_SHA1_32");
+	const char *config=lp_config_get_string(lc->config,"sip","srtp_crypto_suites","AES_CM_128_HMAC_SHA1_80, AES_CM_128_HMAC_SHA1_32, AES_CM_256_HMAC_SHA1_80, AES_CM_256_HMAC_SHA1_32");
 	char *tmp=ms_strdup(config);
 	char *sep;
 	char *pos;
-- 
2.3.2

